Missing Number in a sorted Array.py

```diff
@@ -0,0 +1,16 @@
+# Time Complexity : O(logN)
+#  Space Complexity :O(1)
+def Search(arr, size):
```

Comment:
This is a good solution, also try to make use of index to find the missing number. See the relation between index and missing number!!